lib/internal/Magento/Framework/Stdlib/DateTime/DateTime.php

```diff
@@ -126,9 +126,11 @@ public function gmtTimestamp($input = null)
      * Input date must be in GMT timezone
      *
      * @param  int|string $input date in GMT timezone
+     * @param  null|string $format
+     *
      * @return int
      */
-    public function timestamp($input = null)
+    public function timestamp($input = null, $format = null)
```

Comment:
Unfortunately, we are not able to change the public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov I've made an update to preserve backward compatibility